cmd/root.go

```diff
@@ -135,6 +135,8 @@ or ./scorecard --{npm,pypi,rubgems}=<package_name> [--checks=check1,...] [--show
 		githubClient := github.NewClient(httpClient)
 		graphClient := githubv4.NewClient(httpClient)
 		repoClient := githubrepo.CreateGithubRepoClient(ctx, githubClient)
+		defer repoClient.Close()
```

Comment:
The error gets discarded and never checked right?

Reply:
Yes, the error is never checked in the current implementation.